src/main/kotlin/frc/robot/RobotContainer.kt

```diff
@@ -47,6 +54,9 @@ object RobotContainer {
             }, swerveDrive)
                 .ignoringDisable(true)
         )
+        driverController().a().onTrue(elevator.reset())
+        driverController().x().whileTrue(elevator.setPosition(Units.Centimeter.of(50.0)))
```

Comment:
Should be `onTrue`

Reply:
fe1d3e851aa69c2b022b3914d41c3d3886fe3c5a

Reply:
fe1d3e851aa69c2b022b3914d41c3d3886fe3c5a